target/linux/ramips/image/mt7621.mk

```diff
@@ -955,6 +973,7 @@ define Device/xiaomi_redmi-router-ac2100
   IMAGE/sysupgrade.bin := sysupgrade-tar | append-metadata
   DEVICE_VENDOR := Xiaomi
   DEVICE_MODEL := Redmi Router AC2100
+  DEVICE_MODEL := DEVICE_TITLE := Xiaomi Redmi Router AC2100 (White, 6-Antenna)
```

Comment:
Probably You'll ask why I announced them with their visual definition at menuconfig.
Because there is huge mess in forum topic about white and black editions.
I though this will be better for users which not familier with openwrt and these devices.
And will stop further mess in github, forum, wiki.